### ISSUE TITLE
View/readonly mode for text/markdown even for remote files

### ISSUE THREAD

**Issue by l-mb** [nextcloud/android#10672]
### Is your feature request related to a problem? Please describe.
I create notes and text files via both the web frontend and using my text editor in the synced directories. When I want to view them on my mobile device, they are always opened in edit mode. This easily creates unwanted modifications, but worse, a conflict between multiple collaborators or even multiple of my own devices, unnecessarily.

### Describe the solution you'd like

I want the option to open files in a view-only/read-only mode, which shows markdown formatting and so on, but does not modify the file in any way.

I would like this to be the default (or at least an option to make it the default) when opening a file, since I more often want to view files on the go rather than edit them.

The file should automatically refresh.

This *is* what happens when the file has been downloaded to the local device, so it might be an easy fix?

### Describe alternatives you've considered
While better conflict resolution and merging for text files would also be ways that would alleviate the situation and be beneficial (similar to a lightweight etherpad), the option to only view files seems fairly straightforward.

**Comment by svenb1234** [comment]
If you download the file from within the android client and open it afterwards, it does open in "read mode". Maybe that's a workaround.

**Comment by l-mb** [comment]
> If you download the file from within the android client and open it afterwards, it does open in "read mode". Maybe that's a workaround.

Yes, I noted this in my original post. I would like (at least the option of) this to happen even for files that are still remote and not local.
